.changeset/selfish-bananas-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sigle/app': minor
+---
+
+Create UI with dummy data for referrers on analytics page.
```

Comment:
For changes behind a flag that users can't use directly can we start the message and prefix with (Beta) (so they are sorted in the changelog) and always mark them as patch? Are you fine with this idea?

Reply:
Okay yeah that sounds good - will update it :)